### ISSUE TITLE
utilsScriptDeferred is resolved too early for instances after the first

### ISSUE THREAD

**Issue by molnarm** [jackocnr/intl-tel-input#719]
### Steps to reproduce
1.  Create multiple instances of the phone input, with `utilsScript` specified

### Expected behaviour
Both deferreds returned from creation should be resolved only when the AJAX call to load the script finished.

### Actual behaviour
The deferred for the first instance is resolved only after the utils script is loaded, but the deferred of the second one gets resolved immediately.

### Initialisation options
Set `utilsScript` to a valid value.

### Explanation

This seems to be a regression of #303 , the description there is correct again: in `loadUtils`, there's a check for whether the script loading has _started_, and if that's true, any subsequent calls will get resolved immediately, instead of waiting for completion.

A quick workaround for only this scenario is to _not_ pass the `utilsScriptDeferreds` of instances when calling `loadUtils` from `_initRequests`, because these will be resolved anyway when `handleUtils` is called, but this is not a complete fix (manual calls to `loadUtils` would still work incorrectly).

If you accept pull requests, I created a test for this (failing now, passing with the workaround) [here](https://github.com/jackocnr/intl-tel-input/compare/master...molnarm:concurrency).

**Comment by jackocnr** [comment]
Thanks for reporting this, and writing the test. Yes I welcome pull requests, tho sadly I haven't had enough time to review them in a while. I promise I will get around to it at some point tho!

**Comment by sentiasa** [comment]
```
$.get( "https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/12.1.12/js/utils.js" ).done(function(data) {
            console.log(data);
       $("#number").intlTelInput({
                   initialCountry: "us",
                   utilsScript: data
        }); 
});
```
Is this the right way?

**Comment by jackocnr** [comment]
Fixed by https://github.com/jackocnr/intl-tel-input/commit/295aa0d468b1d7cfcfb184b1d24fe298a37ac6cd

Let me know if you have any other issues tho.
